app/client/src/PluginActionEditor/components/PluginActionResponse/components/QueryResponseTab/QueryResponseTab.tsx

```
@@ -0,0 +1,480 @@
+          },
+        ];
+      }
+    } else {
+      //reset error.
+      errorMessage = "";
+      // TODO: Fix this the next time the file is edited
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      output = actionResponse.body as any;
+    }
+
+    if (actionResponse.messages && actionResponse.messages.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)